networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/Eth2GossipTopicFilter.java

```diff
@@ -80,14 +83,18 @@ private void addTopicsForForkDigest(
     for (int i = 0; i < Constants.ATTESTATION_SUBNET_COUNT; i++) {
       topics.add(getAttestationSubnetTopic(forkDigest, i, gossipEncoding));
     }
+    for (int i = 0; i < NetworkConstants.SYNC_COMMITTEE_SUBNET_COUNT; i++) {
+      topics.add(getSyncCommitteeSubnetTopic(forkDigest, i, gossipEncoding));
+    }
```

Comment:
Probably worth adding a test for this similar to the existing `shouldConsiderAllAttestationSubnetsRelevant`.